models/firstBackEndModels.js

```diff
@@ -34,6 +34,27 @@ function grabArticles() {
     .then(({ rows }) => {
       return rows;
     });
-}
+};
 
-module.exports = { getCurrentTopics, getSpecificArticle, grabArticles };
+const getComments = (article_id) => {
+  return db
+    .query(
+      `SELECT * FROM comments
+       WHERE article_id = $1
+       ORDER BY comments.created_at DESC;`,
+      [article_id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
```

Comment:
if rows.length === 0 it could mean a few things like article is not found or that article is found but doesn't have any comments under it